lib/autoload/kuroko2/workflow/task/retry.rb

```diff
@@ -0,0 +1,55 @@
+module Kuroko2
+  module Workflow
+    module Task
+      class Retry < Base
+        def execute
+          retry_option = parse_option(option)
+          token.context['RETRY'] ||= {}
+          node.children.each do |child|
+            token.context['RETRY'][child.path] = {
+              retried_count: 0,
+              count: retry_option['count'].to_i,
+              sleep_time: retry_option['sleep_time'].to_i
+            }
+          end
+
+          :next
+        end
+
+        def validate
+          retry_option = parse_option(option)
+
+          unless retry_option.has_key?("count")
+            raise_assertion_error
```

Comment:
How about setting the message that `count` option is required?

Reply:
I have modified it in 7f8bd57.